.pre-commit-config.yaml

```diff
@@ -28,13 +29,13 @@ repos:
     rev: v0.4.0
     hooks:
       - id: go-fmt
-## Normally we wouldn't need to do a local hook but we need to modify the shell script that gets run to first change directories into the `cli` folder
+  ## Normally we wouldn't need to do a local hook but we need to modify the shell script that gets run to first change directories into the `cli` folder
```

Comment:
can you clarify why we need to cd into cli when the go.mod is in the root folder now?

Reply:
IDK. It may not be necessary any more. This change was an automated format of the file, I didn't actually change any of this behavior. It would take more more time to go in and figure out whether this is necessary or not. Recommend putting this off for now and adding it to #309